### DISCUSSION TITLE
[Feature request]: 在返回菜单添加特殊按键配置的快速切换

### DISCUSSION THREAD

**Discussion by yaole-A** [ClassicOldSong/moonlight-android#43] [Ideas]
### Is your feature request related to a problem? Please describe.

可以参考这个项目（ https://github.com/WACrown/moonlight-android ），返回键菜单直接可以随时切换特殊按键配置，还有我觉得这个里面的可移动按键（按下某个键并移动鼠标光标，类似触摸板，只不过加上一个按下的键值）也可以加上，还有可隐藏键盘十字摇杆（跟随手指按下位置出现，还可以设置出现的区域），还有组合键、需要哪个键就自己新增

### Describe the solution you'd like

![Screenshot_20241021_093929_com limelight unofficial](https://github.com/user-attachments/assets/98086f8a-6f5f-49e7-9505-88dedf898298)
![Screenshot_20241021_093920_com limelight unofficial](https://github.com/user-attachments/assets/9c05813d-85cc-4983-95a5-60b96315bf1c)
![Screenshot_20241021_093911_com limelight unofficial](https://github.com/user-attachments/assets/7514b117-faef-40db-9362-5b99a19ea588)
![Screenshot_20241021_093900_com limelight unofficial](https://github.com/user-attachments/assets/b6dd6e3f-b950-420f-87a4-d7eac3d79dd7)

### Describe alternatives you've considered

这个（ https://github.com/WACrown/moonlight-android ）可以参考

**Comment by ClassicOldSong** [comment]
工程量较大，暂时不考虑实现

未来完全重构时会重新设计这块的实现
